src/web/mjs/connectors/MangaOvh.mjs

```diff
@@ -0,0 +1,96 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MangaOvh extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangaovh';
+        super.label = 'MangaOvh';
+        this.tags = [ 'manga', 'webtoon', 'russian' ];
+        this.url = 'https://manga.ovh';
+        this.api = 'https://api.manga.ovh';
+    }
+
+    async _getMangasFromPage(page) {
+        const requestUrl = new URL('/book', this.api);
+        requestUrl.searchParams.set('type', 'COMIC');
+        requestUrl.searchParams.set('page', String(page));
+
+        const request = new Request(requestUrl, this.requestOptions);
+        const data = await this.fetchJSON(request, 3);
+
+        return data.content.map(item => {
+            return {
+                id: item.id,
+                title: item.name.en || Object.values(item.name).shift(),
+            };
+        });
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+
+        const request = new Request(new URL('/book?type=COMIC', this.api), this.requestOptions);
+        const data = await this.fetchJSON(request, 1);
+
+        for (let page = 0; page <= data.totalPages; page++) {
+            const mangas = await this._getMangasFromPage(page);
+            mangaList.push(...mangas);
+        }
+
+        return mangaList;
+    }
+
+    async _getMangaFromURI(uri) {
+        const slug = uri.pathname.split('/').pop();
+
+        const requestUrl = new URL('/book/' + slug, this.api);
+        requestUrl.searchParams.set('type', 'COMIC');
+
+        const request = new Request(requestUrl, this.requestOptions);
+        const data = await this.fetchJSON(request, 3);
+
+        return new Manga(this, data.id, data.name.en || Object.values(data.name).shift());
+    }
+
+    async _getBranches(manga) {
```

Comment:
Method only has 2 lines of code and is only used in a single place, code may be inlined instead of extracted in separate method